docs/xmlprofiles.rst

```diff
@@ -178,6 +179,35 @@ The XML label ``<transport_descriptors>`` can hold any number of ``<transport_de
 |                               | parameters and options            |                                 |                |
 |                               | (TCP **only**).                   |                                 |                |
 +-------------------------------+-----------------------------------+---------------------------------+----------------+
+| ``<segment_size>``            | Size (in bytes) of the            | ``uint32``                      | 262144         |
+|                               | shared-memory segment.            |                                 |                |
+|                               | (OPTIONAL, SHM **only**).         |                                 |                |
++-------------------------------+-----------------------------------+---------------------------------+----------------+
+| ``<port_queue_capacity>``     | Capacity (in number of messages)  | ``uint32``                      | 512            |
+|                               | available to every Listener       |                                 |                |
+|                               | (OPTIONAL, SHM **only**).         |                                 |                |
++-------------------------------+-----------------------------------+---------------------------------+----------------+
+| ``<port_overflow_policy>``    | Error policy when a messages      | :class:`OverflowPolicy`         | DISCARD        |
```

Comment:
I cannot see documentation about `OverflowPolicy` anywhere

Reply:
I will add a commit with that information.